src/lambda/manageUserStatus.js

```diff
@@ -0,0 +1,72 @@
+// handles request to approve/ reject user awaiting verification or super user status
+
+import connectToDatabase from './database/dbConnection';
+import {
+  approveSuperUser,
+  rejectSuperUser,
+  approveUser,
+  rejectUser,
+  getUser,
+} from './database/queries/user';
+import { status } from '../constants/users';
+
+export async function handler(event, context) {
+  // eslint-disable-next-line no-param-reassign
+  context.callbackWaitsForEmptyEventLoop = false;
+
+  try {
+    await connectToDatabase();
+    const { user, admin, action, userStatus } = JSON.parse(event.body);
+
+    let update;
+    let msg;
+    const adminExists = await getUser(admin);
+    // check if admin ID Error
+    if (!adminExists) {
```

Comment:
You should check for the user role to be sure they are admin. now any user will be treated as an admin

Reply:
@amusameh good point but I'd like to wait for the auth0 PR to be merged in and then create a reusable role check strategy if thats alright. Not only admin users but super users will be able to use this handler so would be good to set up middlewares to check for that throughout the app

Reply:
@dupreesi good idea I agree